### PR TITLE
PEP8ize context.py

### PR DESCRIPTION
**Pull request by ZuluPro**

### CODE REVIEW

File: askbot/context.py [hunk 1]
```diff
@@ -2,11 +2,13 @@
 from the django settings, all parameters from the askbot livesettings
 and the application available for the templates
 """
-import json
 import sys
+try:
+    import simplejson as json
+except ImportError:
+    import json
```
Comment (ZuluPro) [inline askbot/context.py]: I don't know if you still want to have python 2.6 support

Comment (evgenyfadeev) [inline askbot/context.py]: no, not necessary to support 2.6 thanks!

Comment (evgenyfadeev) [inline askbot/context.py]: Please remove that and I'll merge.

Comment (ZuluPro) [inline askbot/context.py]: Done!

**Pull request merged**